docs/src/adoc/index.adoc

```diff
@@ -9318,23 +9318,6 @@ Configure the `maven-compiler-plugin` in your POM:
 * Java -> Compiler
 * Under "Classfile Generation," check the option "Store information about method parameters (usable via reflection)."
 
-=== GraalVM Native Image
```

Comment:
moving this block messes up the diff - was this intentional? seems like unnecessary shuffling.